src/MetaMorpho.sol

```diff
@@ -344,49 +344,47 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     /// `supplied`).
     /// @dev The allocator can withdraw from any market, even if it's not in the withdraw queue, as long as the loan
     /// token of the market is the same as the vault's asset.
-    function reallocate(MarketAllocation[] calldata withdrawn, MarketAllocation[] calldata supplied)
-        external
-        onlyAllocatorRole
-    {
+    function reallocate(MarketAllocation[] calldata totalAllocation) external onlyAllocatorRole {
```

Comment:
```suggestion
    function reallocate(MarketAllocation[] calldata allocation) external onlyAllocatorRole {
```
Couldn't it just be named allocation ?

Reply:
How do name each element of the array then?

Reply:
Oh yes mb